Chad/components showcase

chadcrume opened the pull request.

alvarix commented:
@chadcrume in my navcleanup PR I moved all these styles into global.css
is there a reason why they should live in Layout.astro?

alvarix commented:
@chadcrume 
As I've started creating components my thinking is evolving as so:
Penguin might not have everything we need and at this stage it doesnt seem like a bad idea to mix and match libraries since a lot of them are copy paste. 

The components page seems like it should come into play later - after we experiment with straight html components, decide which ones we like and how to jigger teh styling so they are uniform and themeable, THEN we can convert them to astro components with props and easily output them in multiple places. 

Does this make sense?

chadcrume commented:
@alvarix , I fixed the conflicts.

Sure, styling can be global. I added some to the global. But in cases where they are truly only for that page or component (like styles to wrap components for demo purposes) they might as well stay local.

I don't think it's too early for a components showcase page. We can add any component to it whenever we think it's ready. Why not go ahead with it since it's done now.

chadcrume commented:
Closing to re-open. Git isn't seeing my conflict rez
